filebeat/module/traefik/access/ingest/pipeline.json

```diff
@@ -43,14 +45,14 @@
         {
             "user_agent": {
                 "field": "traefik.access.agent",
-                "target_field": "traefik.access.user_agent",
-                "ignore_failure": true
+                "ignore_failure": true,
+                "target_field": "traefik.access.user_agent"
             }
         },
         {
-            "rename": {
+            "remove": {
```

Comment:
What is the reason we remove this one now?

Reply:
After spending some minutes looking at it without understanding very well why either, I think that in a first look it made me think that it was a typo, because the field is set in the previous rule. Then, in a second look (maybe I couple of days later) I probably saw the `"target_field": "traefik.access.user_agent.original"` and I also thought it was some kind of typo that shouldn't be there and I delete it.

I guess that the approach here would be to not leave it again as a rename to preserve the original `user_agent` extracted from the Grok pattern, right?

Reply:
Yes, please keep this field around